### PR TITLE
Codewoners: adding Trilinos- & MetisApp

### PR DESCRIPTION
**Pull request by philbucher**

### CODE REVIEW

**Review by pooyan-dadvand — APPROVED** [review]
We hope we can still count on the advice and help of @jcotela

**Comment by loumalouomega** [comment]
Can we extend the file for other apps?

**Comment by philbucher** [comment]
I would say yes, but I still consider it as experimental bcs I am not sure if/when we want to enable the "review from Codeowners"
I would add step by step to it, not all at once

**Pull request merged**
